scenarios/02_along_the_way.cfg

```
@@ -0,0 +1,651 @@
+#textdomain wesnoth-merfolk_campaign
+
+[scenario]
+    id=02_along_the_way
+    name= _ "Along the Way"
+    map_file=02_along_the_way.map 
+    next_scenario=03_the_siege_of_jotha
+
+    {INTRO_AND_SCENARIO_MUSIC frantic.ogg loyalists.ogg}
+    {EXTRA_SCENARIO_MUSIC vengeful.ogg}
+    {EXTRA_SCENARIO_MUSIC transience.ogg}
+
+    {DEFAULT_SCHEDULE_DUSK}
+
+    ## SotA kind of story text goes like this. Works on any resolutions I tried.
+
+    [story]
+        [part]
+            story= _ "With the last naga defeated, Danielle and her party started their journey back east, to Elefas."
```

```suggestion
            story= _ "With the of the last naga defeated, Danielle and her retinue started their journey northeast, back to Elefas."
```

If the naga are moving north to get to Elefas, it would not make sense to say the Danielle's retinue is moving west to east. There needs to be a component of them moving north.